nixos/modules/services/audio/goxlr-utility.nix

```diff
@@ -0,0 +1,48 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.goxlr-utility;
+in
+
+with lib;
```

Comment:
I personally dislike this pattern, it's very confusing to read modules written with this. Could you only inherit the functions you need with `inherit (lib) <functions>;`

Reply:
Valid point, looks better now.